CODE_OF_CONDUCT.md

```diff
@@ -1,83 +1,8 @@
-Code of Conduct
-===============
+# Code of Conduct
 
-Our Pledge
-----------
+This project follows a [Code of Conduct][code_of_conduct] in order to ensure an open and welcoming environment.
+Please read the full text for understanding the accepted and unaccepted behavior.
```

Comment:
behaviors (missing "s"), isn't it?

Reply:
Used generally, behavior is correct in this way. It's used the same way you'd use "walking" or "sitting", I think.